Backend/src/types/node.py

```diff
@@ -6,11 +6,13 @@ class Node(BaseModel):
     Datamodel for a node
     """
 
-    id: str | None = None
+    id: str | None=No
```

Comment:
should be id: str | None = None and the "cluster" field is not used in our new model so it should be removed.